packages/framework/client-logger/external-telemetry/src/factory/index.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IContainer } from "@fluidframework/container-definitions";
+import { ContainerTelemetryManager } from "../container/index.js";
+import { ContainerEventTelemetryProducer } from "../container/telemetryProducer.js";
+import type { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { AppInsightsTelemetryConsumer, type ITelemetryConsumer } from "../common/index.js";
+
+/**
+ * Config object for creating one or more telemetry managers covering differents scopes of the Fluid Framework
+ *
+ * @beta
+ */
+export interface TelemetryManagerConfig {
+	/**
+	 * By providing this attribute, container related external telemetry will be send to any provided ITelemetryConsumer(s)
+	 */
+	container: IContainer;
+	/**
+	 * Consumers for produced external telemetry.
+	 */
+	consumers: ITelemetryConsumer[];
+}
+
+/**
+ * Creates an external telemetry consumer that will send telemetry to Azure Application Insights
+ *
+ * @param client - An instance of an Azure Application Insights client {@link @microsoft/applicationinsights-web#ApplicationInsights}
+ * The App Insights instance must be initialed before being provided which can be done via {@link @microsoft/applicationinsights-web#ApplicationInsights.loadAppInsights}
+ * `applicationInsightsClient.loadAppInsights(); `
```

Comment:
See if you can confirm that writing it like this makes the markdown inline code into a link like I think it should.
```suggestion
 * The App Insights instance must be initialized before being provided, which can be done via {@link @microsoft/applicationinsights-web#ApplicationInsights.loadAppInsights | `applicationInsightsClient.loadAppInsights();` }
 * 
```

Reply:
@Josmithr might now if it's supported

Reply:
I don't know if the back-ticks inside the link text work, but the `{@link target | Alias text}` pattern is correct for aliasing the link text.

Reply:
I couldn't get the alias to load but its fine I'm just removing the little code snippet since we're linking to the function anyways.